packages/lighthouse-spa/src/spaship.yaml

```diff
@@ -0,0 +1,6 @@
+# SPAship config file
+# generated by @spaship/common.config v0.14.0
+
+name: lighthouse-spa
+path: /lighthouse-spa
```

Comment:
Can we make it `/lighthouse` instead of the extra `-spa`?

Reply:
we already have the redirection about the ci server for now let keep it like this. we can dicusss about the path in grooming call.